design_introspection-plugin/get_pins.cc

```diff
@@ -0,0 +1,60 @@
+#include "get_pins.h"
+
+USING_YOSYS_NAMESPACE
+
+std::string GetPins::TypeName() { return "pin"; }
+
+std::string GetPins::SelectionType() { return "c"; }
+
+void GetPins::ExecuteSelection([[gnu::unused]] RTLIL::Design* design,
```

Comment:
Shouldn't it be more roboust to cast these unused args to void (like `(void)design;`) instead of using `[[gnu::unused]]` ?

Reply:
Not sure, which one is more robust, but `(void)design` is more versatile so I will take the suggestion.